ui/app/reports/controllers/reportsController.js

```diff
@@ -2,29 +2,56 @@
 
 angular.module('bahmni.reports')
     .controller('ReportsController', ['$scope', 'appService', 'reportService', 'FileUploader', 'messagingService', 'spinner', '$rootScope', 'auditLogService', function ($scope, appService, reportService, FileUploader, messagingService, spinner, $rootScope, auditLogService) {
+        const format = _.values(reportService.getAvailableFormats());
+        const dateRange = _.values(reportService.getAvailableDateRange());
+
         $scope.uploader = new FileUploader({
             url: Bahmni.Common.Constants.uploadReportTemplateUrl,
             removeAfterUpload: true,
             autoUpload: true
         });
-
         $scope.uploader.onSuccessItem = function (fileItem, response) {
             fileItem.report.reportTemplateLocation = response;
         };
 
-        $rootScope.default = _.isUndefined($rootScope.default) ? {reportsRequiringDateRange: {}, reportsNotRequiringDateRange: {}} : $rootScope.default;
+        $rootScope.default = _.isUndefined($rootScope.default) ? {
+            reportsRequiringDateRange: {
+                responseType: format[1],
+                dateRangeType: dateRange[0],
+                startDate: dateRange[0],
+                stopDate: dateRange[0],
+                report: {
+                    responseType: format[1]
+                }
+            },
+            reportsNotRequiringDateRange: {}
+        } : $rootScope.default;
         $scope.reportsDefined = true;
         $scope.enableReportQueue = appService.getAppDescriptor().getConfigValue("enableReportQueue");
-
         $scope.setDefault = function (item, header) {
             var setToChange = header === 'reportsRequiringDateRange' ? $rootScope.reportsRequiringDateRange : $rootScope.reportsNotRequiringDateRange;
             setToChange.forEach(function (report) {
-                report[item] = $rootScope.default[header][item];
+                if (item == 'dateRangeType') {
+                    $rootScope.default.reportsRequiringDateRange.startDate = $rootScope.default[header][item];
+                    $rootScope.default.reportsRequiringDateRange.stopDate = dateRange[0];
+                    report['startDate'] = $rootScope.default[header][item];
+                    report['stopDate'] = dateRange[0];
+                }
+                else if ($rootScope.default[header][item] === undefined) {
+                    $rootScope.reportsRequiringDateRange.forEach(report => {
```

Comment:
This line is throwing the error. I think, arrow functions are not supported in Angular Js. Try removing arrow function. This should probably fix the issue.